stdlib/runtime/gc.gr

```diff
@@ -74,7 +76,7 @@ let rec decRef = (userPtr: WasmI32, ignoreZeros: Bool) => {
       if (ignoreZeros) {
         userPtr
       } else {
-        fail "decRef called when reference count was zero."
+        throw DecRefError
```

Comment:
Does `DecRefError` not need a printer?

Reply:
I don't think so. This isn't actually an error that ever happens outside of GC debug mode. Since this module gets loaded for every program, I figure we can save on cycles by not registering a printer for it, and we can register one locally if we really want while debugging the GC.